src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/project.json

```diff
@@ -23,7 +23,7 @@
     }
   },
   "dependencies": {
-    "Microsoft.DotNet.InternalAbstractions": "1.0.0",
+    "Microsoft.DotNet.InternalAbstractions": "1.0.1-*",
```

Comment:
:up: :date:
DependencyModel gets brought in by CLI Utils, ProjectModel, ProjectModel.Workspaces. 
So had to pin all of those.